co-simulation tool/fed/mosaic-carla/src/main/java/org/eclipse/mosaic/fed/carla/ambassador/CarlaAmbassador.java

```diff
@@ -0,0 +1,490 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
```

Comment:
If this is a new class, modify the Copyright header.

Reply:
The Copyright header has been updated.